packages/epics/src/organisation/components/list-proposals.tsx

```diff
@@ -0,0 +1,190 @@
+'use client';
+import { useState } from 'react';
+import { Text } from '@radix-ui/themes';
+import { PlusIcon } from '@radix-ui/react-icons';
+import { TabsContent, Tabs, TabsList, TabsTrigger } from '@hypha-platform/ui/server';
+import { Button, FilterMenu } from '@hypha-platform/ui';
+import { CardProposal } from './card-proposal';
+
+type ListProposalsProps = Record<string, never>;
+
+type OptionType = {
+  label: string,
+  value: string
+}
+
+type FilterType = {
+  value: string,
+  options: OptionType[]
+}
+
+const proposalsfilterSettings: FilterType = {
+  value: 'most-recent',
+  options: [
+    { label: 'All', value: 'all' },
+    { label: 'Most recent', value: 'most-recent' }
+  ],
+};
+
+const dummyData = {
+  proposalsCount: 32,
+  proposals: [
+    {
+      title: 'Proposal Title',
+      creator: {
+        avatar: "https://images.unsplash.com/photo-1544005313-94ddf0286df2?&w=64&h=64&dpr=2&q=70&crop=faces&fit=crop",
+        name: 'Jane',
+        surname: 'Doe'
+      },
+      commitment: 50,
+      status: 'active',
+    },
+    {
+      title: 'Proposal Title',
+      creator: {
+        avatar: "https://images.unsplash.com/photo-1544005313-94ddf0286df2?&w=64&h=64&dpr=2&q=70&crop=faces&fit=crop",
+        name: 'Jane',
+        surname: 'Doe'
+      },
+      commitment: 50,
+      status: 'voting',
+    },
+    {
+      title: 'Proposal Title',
+      creator: {
+        avatar: "https://images.unsplash.com/photo-1544005313-94ddf0286df2?&w=64&h=64&dpr=2&q=70&crop=faces&fit=crop",
+        name: 'Jane',
+        surname: 'Doe'
+      },
+      commitment: 50,
+      status: 'completed',
+    },
+    {
+      title: 'Proposal Title',
+      creator: {
+        avatar: "https://images.unsplash.com/photo-1544005313-94ddf0286df2?&w=64&h=64&dpr=2&q=70&crop=faces&fit=crop",
+        name: 'Jane',
+        surname: 'Doe'
+      },
+      commitment: 50,
+      status: 'rejected',
+    }
+  ]
+}
+
+export const ListProposals: React.FC<ListProposalsProps> = () => {
```

Comment:
I'd say, the `ListProposals` component is part of another epic e.g. `agreements` or `proposals`